release/rllib_tests/app_config.yaml

```diff
@@ -13,6 +13,12 @@ debian_packages:
 
 python:
   pip_packages:
+  ## These dependencies should be handled by rllib-requirements.txt
+  ## and removed here
+  - gymnasium[atari,mujoco]==0.28.1
```

Comment:
This needs to be changed here https://github.com/ray-project/ray/blob/master/release/ray_release/byod/requirements_ml_byod_3.8.in#L16.

Then compile with bazel run //release:requirements_ml_byod_3.8.update

Release tests are no longer using these app_config.yaml files.

Reply:
Let me know if you run into issues doing that, I can put up the PR for you as well